src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableClient.cs

```diff
@@ -225,7 +225,7 @@ Task IDurableEntityClient.SignalEntityAsync(EntityId entityId, string operationN
                     ConnectionName = connectionName,
                 };
 
-                TaskHubClient taskHubClient = ((DurableClient)this.config.GetClient(attribute)).client;
+                var taskHubClient = (DurableClient)this.config.GetClient(attribute);
```

Comment:
nit: this variable is now incorrectly named, as it is no longer a `TaskHubClient`, but a `DurableClient`